beets/autotag/mb.py

```diff
@@ -265,6 +265,7 @@ def album_info(release):
             )
             ti.disctitle = disctitle
             ti.media = format
+            ti.alt_track_no = track['number']
```

Comment:
To complete my understanding, what does this `number` field usually hold? Is it typically a number, but sometimes something else?